facdb/sql/_qaqc.sql

```diff
@@ -1,3 +1,9 @@
+-- Create full table of previous data (all records and all geometries that exist)
+ALTER TABLE dcp_facilities_with_unmapped DROP COLUMN IF EXISTS wkb_geometry;
```

Comment:
Is this a fail safe addition, just in case any future csv as the `wkb_geometry` by mistake?